week1-webclient_ui.md

```diff
@@ -0,0 +1,101 @@
+# Codess in the Classroom - Week 1: The Web Client
+
+## Building the Pet Shelter UI
+
+There are several UI frameworks that can be used to have a good-looking UI with React without
+the need of manually writing style instructions.
+For instance, one can use [Bootstrap][bootstrap], [Material-UI][material-ui] or [Office UI Fabric][office-ui-fabric]. For this project, we suggest using Bootstrap, through its most popular React bindings: [react-boostrap][react-bootstrap].
+
+1. Run `npm install react-bootstrap @types/react-bootstrap` to install the `react-boostrap` library.
+2. Reference the bootstrap css file in the HTML index file located in `public/index.html` as follows.
+
+        <link
+            rel="stylesheet"
+            href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+            integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+            crossorigin="anonymous"
+        />
+
+3. Use the Bootstrap component library to build a simple view for all the animals hosted by the shelter. Here are some useful layouts and components to consider:  
+    - Grid layout system: https://react-bootstrap.github.io/layout/grid/
+    - Cards: https://react-bootstrap.github.io/components/cards/
+    - Card Layouts: https://react-bootstrap.github.io/components/cards/#card-layout
+
+
+An example of the view is shown in the image below - though, by all means, feel free to come up with anything that looks different.
+
+<p align="center"> 
+<img src="https://user-images.githubusercontent.com/1350095/56684256-48f43600-66c7-11e9-9e9b-648d118fb76f.jpg" alt="smaple list view UI" width="400"/>
+</p>
+
```

Comment:
Could we add a sentence in here saying explicitly something like: "If you aren't familiar with React components or how to use them to build a UI, you can view the more detailed instructions below."?

Reply:
yup makes sense. And a general intro to React should be covered by  https://github.com/codessintheclassroom/classroom-material/issues/8